arch/arm/core/cortex_m/mpu/Kconfig

```diff
@@ -17,6 +17,14 @@ config ARM_MPU
 	  Note that NXP MPU as well as ARMv8-M MPU does not require MPU regions
 	  to have power-of-two alignment for base address and region size.
 
+config ARM_MPU_REGION_MIN_ALIGN_AND_SIZE
```

Comment:
It's nice to remove the hardcoded value from the code, but I wonder if Kconfig is the right place for this value. Wouldn't it be better if this is put in one of the _internal.h files instead?

Reply:
This was discussed with @SebastianBoe . The reason for a Kconfig _hidden_ symbol, was to make both the C code and the linker scripts use a common reference for this fundamental MPU property, instead of having linker scripts including C headers. This was seen as a cleaner option.

Reply:
Ok